lib/logzio-nodejs.js

```diff
@@ -78,6 +79,17 @@ class LogzioLogger {
 
         this.protocol = protocol;
         this._setProtocol(port);
+        this.url = `${this.protocol}://${this.host}:${this.port}?token=${this.token}`;
+
+		this.axiosInstance = axiosIn;
+        this.axiosInstance.defaults.headers.post = {
```

Comment:
what happened to your indentation
dont u work with vscode? pls enable the linting